includes/class-wc-stripe-order-helper.php

```
@@ -652,6 +654,58 @@ public function update_stripe_mandate_id( ?WC_Order $order = null, string $manda
+			)
+		);
+
+		if ( ! empty( $order_id ) ) {
+			$order = wc_get_order( $order_id );
+			if ( $order && $order->get_status() !== OrderStatus::TRASH ) {
+				return $order;
+			}
+		}
+
+		return null;
+	}
```

PHPMD (2.15.0)

[error] The method get_order_by_mandate_id is not named in camelCase. (undefined)

(CamelCaseMethodName)